recipe/patches/0008-Add-osx-64-and-linux-ppc64le-as-supported-wheel-tags.patch

```diff
@@ -20,14 +20,14 @@ index c40bd66ac..809b1faf0 100644
 +    flag_values = {
 +        "//toolchain:release": "True",
 +    },
-+    values = {"cpu": "linux-ppc64le"},
++    values = {"cpu": "linux-ppc"},
```

Comment:
I see https://github.com/conda-forge/bazel-toolchain-feedstock/pull/23. Could you explain for posterity where that `ppc` value comes from in protobuf, i.e. where we need to look to know where this value comes from?

Reply:
`ppc` is the old/deprecated value for `ppc64le`. `rules_python` still uses this, and we patch ppc64le support into protobuf; thus we need to follow there what `rules_python` expects.